src/troute-routing/troute/routing/diffusive_utils.py

```diff
@@ -1165,8 +1136,8 @@ def diffusive_input_data_v02(
     para_ar_g     = np.zeros(paradim)
     para_ar_g[0]  = 0.95    # Courant number (default: 0.95)
     para_ar_g[1]  = 0.5     # lower limit of celerity (default: 0.5)
-    para_ar_g[2]  = 50.0    # lower limit of diffusivity (default: 50)
-    para_ar_g[3]  = 5000.0  # upper limit of diffusivity (default: 1000)
+    para_ar_g[2]  = 10.0    # lower limit of diffusivity (default: 50)
+    para_ar_g[3]  = 40000.0  # upper limit of diffusivity (default: 1000)
```

Comment:
If these are the default values, then the comments should be updated appropriately...

Reply:
Changed. Thanks.